.github/workflows/run-acceptance-tests.yml

```diff
@@ -252,6 +252,8 @@ jobs:
       with:
         ref: ${{ env.PR_COMMIT_SHA }}
         submodules: true
+    - name: Init upstream
```

Comment:
It seems reasonable that leaving this out could upset things; but could you record in the commit message what fails and how this fixes it, please.

I think you'll need to replicate this change in the other places this test runs, too.

Reply:
Turns out this wasn't necessary -- updating the shim/go.mod was enough.